### PR TITLE
Add simple `MLflowCallback` example

### LINKED ISSUE #56: Add `MLflowCallback` example
## Motivation

This repo does not provide a minimal example of [`MLflowCallback`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.integration.MLflowCallback.html). There exist two MLflow examples as follows:

- [`mlflow/keras_mlflow.py`](https://github.com/optuna/optuna-examples/blob/e68e86d5c7fd91727da869d9708a770757150da9/mlflow/keras_mlflow.py): using pure MLflow, not `MLflowCallback`.
- [`kubernetes/mlflow/pytorch_lightning_distributed.py`](https://github.com/optuna/optuna-examples/blob/6a6b20ad634627eebb3e7e104f73b70b45c6e624/kubernetes/mlflow/pytorch_lightning_distributed.py): using `MLflowCallback`, but it looks complicated.

I think it would be great to provide such a minimal example.

## Description

Please add mlflow callback example similar way to https://github.com/optuna/optuna-examples/issues/41.

### PR DESCRIPTION
**Pull request by xadrianzetx**
## Motivation

Previously there was no minimal example of  `MLflowCallback`. Additionally, existing keras example did not use `MLflowCallback`, as discovered in https://github.com/optuna/optuna-examples/issues/41#issuecomment-920145258. This PR fixes both those issues. Resolves #56.

## Description of the changes

* Add minimal `MLflowCallback` example
* Use `MLflowCallback` in keras example
* Update CI and README

### CODE REVIEW

File: mlflow/README.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
 
 ![mlflow-ui](https://user-images.githubusercontent.com/17039389/70850501-4cdefd80-1ece-11ea-9018-e47363c81f08.gif)
 
+
+### mlflow_simple.py
```
Comment (nzw0301) [inline mlflow/README.md:6]: [nit]
how about adding a hyperlink to hte file?
```suggestion
### [mlflow_simple.py](mlflow_simple.py)
```

File: mlflow/README.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+In this example we optimize random forest classifier for the Iris dataset. All
+hyperparameters and metrics will be logged to MLflow via integration callback.
+
+You can run this example as follows:
+
+```
+$ python mlflow_simple.py
+```
+
+### keras_mlflow.py
```
Comment (nzw0301) [inline mlflow/README.md:20]: ditto
```suggestion
### [keras_mlflow.py](keras_mlflow.py)
```

**Review by nzw0301 — COMMENTED** [review]
Let me leave minor comments.

**Comment by nzw0301** [comment]
Like always, thank you for your sold contributions! 

I think the two examples code are almost the same except for the objective function. I suppose that rename `keras_mlflow.py` with `mlflow_simple.py` with the proposed modification in this PR is enough for simplicity.  Sorry, I should have discussed more when I had created the issue. However, I'm not sure about the backward compatibility-ish policy in this repo. So I would like to the senior member's comment.

cc: @HideakiImamura @hvy @toshihikoyanase

**Comment by xadrianzetx** [comment]
Sorry, my mistake. I can close this PR and open new one with smaller scope if it's ok with you.

**Pull request closed without merging**